docs/data/pages.ts

```diff
@@ -4,22 +4,75 @@ const pages: MuiPage[] = [
   {
     pathname: '/toolpad/getting-started',
     title: 'Getting started',
-    icon: 'CreateIcon',
+    icon: 'DescriptionIcon',
     children: [
-      { pathname: '/toolpad/getting-started/setup' },
-      { pathname: '/toolpad/getting-started/tutorial', title: 'Tutorial' },
+      { pathname: '/toolpad/getting-started/introduction', title: 'Introduction' },
+      { pathname: '/toolpad/getting-started/quickstart', title: 'Quickstart' },
     ],
   },
   {
-    pathname: '/toolpad/data-fetching',
+    pathname: '/toolpad/connecting-to-data-sources',
+    title: 'Connecting to data sources',
     icon: 'TableViewIcon',
     children: [
-      { pathname: '/toolpad/data-fetching', title: 'Overview' },
-      { pathname: '/toolpad/data-fetching/rest', title: 'REST API' },
-      { pathname: '/toolpad/data-fetching/google-sheets', title: 'Google Sheets' },
-      { pathname: '/toolpad/data-fetching/queries' },
+      {
+        pathname: '/toolpad/connecting-to-data-sources/queries',
+        title: 'Queries',
+      },
+      {
+        pathname: '/toolpad/connecting-to-data-sources/connections',
+        title: 'Connections',
+      },
+      {
+        pathname: '/toolpad/connecting-to-data-sources/fetch',
+        title: 'Fetch type',
```

Comment:
Not sure about the "type" nomenclature. But perhaps "Fetch datasource" is too long?

Reply:
Gonna update it all across docs 👍